families/burrow_evm/src/protos/evm.proto

```diff
@@ -32,17 +34,73 @@ message EvmStateAccount {
 
     // In a future iteration, a public_key field may be added to validate
     // message calls have been sent by an authorized account.
-    }
+}
 
 message EvmStorage {
     bytes key = 1;
     bytes value = 2;
 }
 
-// This is the payload delivered to the TP and follows the transaction
+// -- Transactions --
+
+message EvmTransaction {
+    enum TransactionType {
+      DEFAULT = 0;
+      CREATE_EXTERNAL_ACCOUNT = 1;
+      CREATE_CONTRACT_ACCOUNT = 2;
+      MESSAGE_CALL = 3;
+    }
+    TransactionType transaction_type = 1;
+
+    // To eliminate the need for two deserialization steps, all types of
+    // transactions are included as fields, but only the field indicated by the
+    // transaction type should be set.
+    CreateExternalAccountTxn create_external_account = 2;
```

Comment:
Ah, ok, fair enough.

I'm amused by this quote, though: 

> All that said, the reason that this functionality is not included in the Protocol Buffer library is because we have never had a use for it inside Google.